src/inc/utilcode.h

```diff
@@ -5750,4 +5750,21 @@ extern SpinConstants g_SpinConstants;
 
 // ======================================================================================
 
+#ifdef FEATURE_GDBJIT
```

Comment:
I would prefer creating a separate header src/vm/gdbjit.h instead of sticking this stuff into the utilcode.h.